src/main/kotlin/io/emeraldpay/dshackle/upstream/ethereum/WsConnectionMultiPool.kt

```diff
@@ -50,6 +53,8 @@ class WsConnectionMultiPool(
     private var adjustLock = ReentrantReadWriteLock()
     private val index = AtomicInteger(0)
     private var connIndex = 0
+    private val connectionInfo = Sinks.many().unicast().onBackpressureError<WsConnection.ConnectionInfo>()
```

Comment:
unicast? Looks like it should be multicast

Reply:
You're right, thanks!